### PR TITLE
Add buzzfeed sso support

### PR DESCRIPTION
**Pull request by Uxio0**

### CODE REVIEW

**Comment by Uxio0** [comment]
Replaces #551

File: config/settings/base.py [hunk 1]
```diff
@@ -196,7 +196,8 @@
 ADMIN_URL = "admin/"
 # https://docs.djangoproject.com/en/dev/ref/settings/#admins
 ADMINS = [
-    ("""Gnosis""", "dev@gnosis.pm"),
+    ("uxio.fuentefria", "uxio.fuentefria@gnosis.pm"),
+    ("richard", "richard.meissner@gnosis.pm"),
```
Comment (fmrsabino) [inline config/settings/base.py:200]: I think this is better to be set on an environment variable.

With the amount of public usage by different orgs this is definitely something that admins would configure.

**Review by fmrsabino — CHANGES_REQUESTED** [review]

File: config/settings/base.py (hunk 1 above)
Comment (Uxio0) [reply config/settings/base.py:200]: You are right, what I would do it's make it an environment variable but leave ourselves as the default ones:
- We already had gnosis configuration in place in that variable
- Removing us would imply modifying 12 deployments in production + 6 in staging

Comment (Uxio0) [reply config/settings/base.py]: Done 👍🏻

**Review by fmrsabino — APPROVED** [review]

File: safe_transaction_service/history/views.py [hunk 2]
```diff
@@ -66,13 +66,20 @@ class AboutView(APIView):
 
     renderer_classes = (JSONRenderer,)
 
-    @method_decorator(cache_page(60 * 60))  # 1 hour
+    # @method_decorator(cache_page(60 * 60))  # 1 hour
```
Comment (fmrsabino) [inline safe_transaction_service/history/views.py:69]: I only saw this now but was this intended? @Uxio0

Comment (Uxio0) [reply safe_transaction_service/history/views.py:69]: No. I disabled cache to test the headers

Comment (Uxio0) [reply safe_transaction_service/history/views.py:69]: Thanks for noticing

Comment (Uxio0) [reply safe_transaction_service/history/views.py:69]: https://github.com/gnosis/safe-transaction-service/pull/757

**Pull request merged**